src/pages/Offer/sections/Offer.tsx

```diff
@@ -156,9 +152,12 @@ export const Offer: React.SFC<Props> = ({ signButtonVisibility, offer }) => (
           <HeaderBackground>
             <HeaderWrapper>
               <Header>
-                <TranslationsConsumer textKey="OFFER_HEADER">
+                <TranslationsPlaceholderConsumer
+                  textKey="OFFER_HEADER"
+                  replacements={{ name: offer.member.firstName }}
```

Comment:
could we name this replacement `firstName`? i think that's what we use in other places